### PR TITLE
allow 0 in all [cocoapods] download tests

### PR DESCRIPTION
**Pull request by chris48s**
It appears this does not only happen on the weekly stats.

refs https://github.com/badges/shields/pull/1713

### CODE REVIEW

File: services/cocoapods/cocoapods.tester.js [hunk 1]
```diff
@@ -4,14 +4,21 @@ const Joi = require('joi');
...
 const isPlatform = Joi.string().regex(/^(osx|ios|tvos|watchos)( \| (osx|ios|tvos|watchos))*$/);
 
+// these are deliberately not isMetricOverTimePeriod due to
+// https://github.com/CocoaPods/cocoapods.org/issues/348
+const isMetricOverTimePeriodAllowZero = Joi
+  .string()
+  .regex(/^(0|[1-9][0-9]*)[kMGTPEZY]?\/(year|month|4 weeks|week|day)$/);
+const isMetricAllowZero = Joi
+  .string()
+  .regex(/^(0|[1-9][0-9]*)[kMGTPEZY]?$/);
```
Comment (RedSparr0w) [inline services/cocoapods/cocoapods.tester.js]: Would it be worth adding these to `test-validators`?

Comment (chris48s) [reply services/cocoapods/cocoapods.tester.js]: Normally we only do this once more than one suite of tests use a validator, but I don't have particularly strong opinions on it..

Comment (RedSparr0w) [reply services/cocoapods/cocoapods.tester.js]: I think `npm`,  `hex.pm`, `packagist`, `source forge`, `DUB`, `package control` and `eclipse marketplace` all use the same style downloads-badge
Although it looks like `npm` is missing test for `weekly/monthly/yearly` stats

Comment (chris48s) [reply services/cocoapods/cocoapods.tester.js]: I think cocoapods is the only service where we want to allow this (unless we hit the same issue with one of those other services). We should continue to expect >0 in other cases as per the discussion in https://github.com/badges/shields/pull/1713#discussion_r192567128

Comment (RedSparr0w) [reply services/cocoapods/cocoapods.tester.js]: No worries, sounds good.
:+1:

**Pull request merged**